apps/src/lib/kits/maker/boards/microBit/MicroBitBoard.js

```diff
@@ -48,6 +48,7 @@ export default class MicroBitBoard extends EventEmitter {
 
     /** @private {MicrobitFirmataClient} serial port controller */
     this.boardClient_ = new MBFirmataWrapper(portType);
+    this.boardClientWithoutPort = null;
```

Comment:
Is there a way we can make this a property of `this.boardClient_`? For example, calling `this.boardClientWithoutPort = this.boardClient_.withoutPort`? Not sure how that would work in `initializeComponents`, but it might make sure these values stay tightly coupled.

Reply:
Notes: chatted about making a `getter` in the wrapper to encapsulate more.

Reply:
I like this idea! Just pushed the refactor.

Reply:
I think we can get rid of this variable now, right?

Reply:
Oh yes! Forgot to go back and delete. Will do!